packages/golden-layout/src/container/ItemContainer.ts

```diff
@@ -31,7 +31,7 @@ export default class ItemContainer<
   _element = $(
     [
       '<div class="lm_item_container">',
-      '<div class="lm_content" tabindex="-1"></div>',
+      '<div class="lm_content" tabIndex="-1"></div>',
```

Comment:
This is `jQuery`, not react, so you still want it to be just [`tabindex`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex). 
```suggestion
      '<div class="lm_content" tabindex="-1"></div>',
```